### PR TITLE
fix(*): move .wizard-title to global.less

### PR DESCRIPTION
**Pull request by AxelPeter**

### CODE REVIEW

File: client/app/css/less/common/global.less [hunk 1]
```diff
@@ -15,6 +15,16 @@ input:focus,
...
 }
 
+.wizard-title {
+    line-height: rem-calc(22.4);
+    margin-top: 0;
+    margin-bottom: 0;
+    color: @oui-color-zodiac;
+    font-size: rem-calc(21);
+    font-weight: 600;
+    letter-spacing: 0.01em;
```
Comment (antleblanc) [inline client/app/css/less/common/global.less]: `.01em` :candy:

File: client/app/css/less/common/global.less [hunk 2]
```diff
@@ -15,6 +15,16 @@ input:focus,
     -moz-outline: none;
 }
 
+.wizard-title {
+    line-height: rem-calc(22.4);
+    margin-top: 0;
+    margin-bottom: 0;
+    color: @oui-color-zodiac;
+    font-size: rem-calc(21);
+    font-weight: 600;
```
Comment (antleblanc) [inline client/app/css/less/common/global.less]: `600` => `@oui-font-semibold`?

File: client/app/css/less/common/global.less (hunk 1 above)
Comment (AxelPeter) [reply client/app/css/less/common/global.less]: Done. It was a copypaste from ovh-ui-kit to replace &:extend(.modal__title).

File: client/app/css/less/common/global.less (hunk 2 above)
Comment (AxelPeter) [reply client/app/css/less/common/global.less]: Done

**Pull request merged**